log4j-core/src/test/java/org/apache/logging/log4j/core/LoggerUpdateTest.java

```diff
@@ -16,11 +16,15 @@
  */
 package org.apache.logging.log4j.core;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertSame;
```

Comment:
static imports go last. We allow wildcard static imports so this change isn't necessary.